clippy_lints/src/types.rs

```diff
@@ -44,9 +44,9 @@ pub struct TypePass;
 ///     values: Vec<Foo>,
 /// }
 /// ```
-declare_lint! {
+declare_clippy_lint! {
     pub BOX_VEC,
-    Warn,
+    complexity,
```

Comment:
perf?

Reply:
both ^^ it's not just slower or requires more memory, it's just weird